python/llm/example/GPU/vLLM-Serving/README.md

```diff
@@ -61,6 +60,7 @@ pip install transformers_stream_generator einops tiktoken
 ```bash
 export USE_XETLA=OFF
 export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```

Comment:
export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=2 in `start-vllm-service.sh`. Should we align this?